bin/start-local-environment.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+## Usage: ./bin/start-local-environment DOCMOSIS_ACCESS_KEY
+##
+## Options:
+##    - DOCMOSIS_ACCESS_KEY: Access key for docmosis development environment.
+##    - DOCMOSIS_TEMPLATES_ENDPOINT_AUTH: Access key for docmosis templates endpoint.
+##
+## Start local environment including idam client setup.
+
+
+# Set variables
+COMPOSE_FILE="docker-compose-dependencies.yml"
+IDAM_URI="http://localhost:5000"
+IDAM_USERNAME="idamOwner@hmcts.net"
+IDAM_PASSWORD="Ref0rmIsFun"
```

Comment:
this is trouble :)
is this for local use or to run on jenkins? it's possible to get these from the vault, saves us having to deal with IA
az keyvault secret show --vault-name em-icp-aat --name redis-password

Reply:
local use, happy to remove